backend/db.json

```diff
@@ -7,5 +7,10 @@
     { "title": "Testimonials" },
     { "title": "blog" },
     { "title": "contact" }
-  ]
+  ],
+  "estimatesection": {
+    "subTitle": "Estimate your Project!",
+    "sectionTitle": "Have Any Project On Mind ?",
+    "btnLabel": "get in touch"
```

Comment:
I think You get confused about the title and subtitle.
- the `API` should be like 
"title": "Have Any Project On Mind?" ==> and just title .. not `sectionTitle`
"subTitle": "Estimate your Project!"
"button": {
"title": "Get Touch",
"url": "/"
}
- please put them at the right place, don't reflect the titles